### PR TITLE
Schnebly/feature/functional calcs

### PR DESCRIPTION
**Pull request by schnebly**
Self-explanatory :) Just a rewrite -- nothing new feature wise.

### CODE REVIEW

File: analyzer-service/calculations.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+
+def loadHistorical(ticker):
+  '''
+  load historical data for class attribute - ticker
+
+  @return bool set to true if functions completes
+  '''
+  client, collection = DBInterface.connect() #need try catch here
+  cursor = collection.find_one({"MetaData.2Symbol" : "{}".format(ticker)})
```
Comment (ayovev) [inline analyzer-service/calculations.py]: This is going to change when we configure pre-insertion key preprocessing (hopefully this week) but I think this is fine for now

File: analyzer-service/app.py [hunk 2]
```diff
@@ -66,51 +66,51 @@ def redeploy():
 @app.route("/analyzer/var/<ticker>")
 def valueAtRisk(ticker):
   '''value at risk'''
-  ticker, data = Calc.loadHistorical(ticker)
-  test = Calc.calculateVar(ticker, data)
+  ticker, data = loadHistorical(ticker)
```
Comment (ayovev) [inline analyzer-service/app.py]: If we're calling the `loadHistorical` method for every route, shouldn't that just be part of the calculation function? Or is there a better way to do this?

File: analyzer-service/calculations.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  pReturns = pd.to_numeric(df.iloc[:, 5]).pct_change()
+
+  # calculate the percent return's mean and s.d.
+  pr_mean = np.mean(pReturns)
+  pr_std = np.std(pReturns)
+
+  # use normal distribution percentile function to get Value at Risk (90, 95, 99)
+  VaR_90 = norm.ppf(.1, pr_mean, pr_std)
+  VaR_95 = norm.ppf(.05, pr_mean, pr_std)
+  VaR_99 = norm.ppf(.01, pr_mean, pr_std)
```
Comment (ayovev) [inline analyzer-service/calculations.py]: Thank you for doing these calculation at different levels

File: analyzer-service/calculations.py (hunk 1 above)
Comment (schnebly) [reply analyzer-service/calculations.py]: Yeah, easy change for when we do that.

File: analyzer-service/app.py (hunk 2 above)
Comment (schnebly) [reply analyzer-service/app.py]: Yes, I am going to create a calculation function that that essentially does all calcs for each ticker in a given list but I wanted to talk to you about the chrontab stuff before doing that.

File: analyzer-service/calculations.py (hunk 3 above)
Comment (schnebly) [reply analyzer-service/calculations.py]: No problem

**Pull request merged**